Samples/CSharp/CSharpSamples.csproj

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Perhaps we could use a newer .NET standard .csproj file.

Reply:
By all means!